js/views/dataset-edit.vue

```diff
@@ -30,12 +30,24 @@ export default {
     },
     methods: {
         save() {
-            const form = this.$refs.form;
+            let form = this.$refs.form;
             if (form.validate()) {
                 this.dataset.update(form.serialize(), (response) => {
-                    this.dataset.on_fetched(response);
+                    this.dataset.on_fetched(response); 
+                    
+                    if(!form.hideNotifications){
+                        this.$dispatch('notify', {
+                            autoclose: true,
+                            title: this._('Changes saved'),
+                            details: this._('Your dataset has been updated.')
+                        });
+                    }
+
                     this.$go({name: 'dataset', params: {oid: this.dataset.id}});
-                }, form.on_error);
+                }, (error)=>{
```

Comment:
Spaces around arrow I think (and maybe drop the `()`)

Reply:
Sorry about « drop the () », I just realized we usually leave them (see response above).